tests/testdata/provider/testdata_pg_bigint_pk.sql

```diff
@@ -104,20 +104,29 @@ INSERT INTO qgis_test.provider_bigint_nonfirst_pk  (zeroth_field, key1, key2, pr
 (1, 2, 3, 4,  400, 'Honey', 'Honey', '4', TIMESTAMP '2021-05-04 13:13:14', '2021-05-04', '13:13:14', '0101000020E610000014AE47E17A5450C03333333333935340')
 ;
 
-/* -- PostgreSQL 12 or later
```

Comment:
please don't drop these!

Reply:
Hi, I've included these in a previous PR because I had started working with tests against partitioned tables, but since working with partitions is transparent to the user, I saw no reason to test against these. These tables are, as of now, dead code, they are not used in any of the tests.

Also, the comment was wrong: these are not for PostgreSQL 12, but also work with 11 (except that the values in the clause FROM x TO y must be in parenthesis).

Do you still want to keep it?

Reply:
Commit 6578c98 addresses the issue of quoting floating point values.